src/controller/python/build-chip-wheel.py

```diff
@@ -66,13 +69,18 @@ def __init__(self, name):
 manifestFile = os.path.abspath(args.manifest)
 buildDir = os.path.abspath(args.build_dir)
 distDir = os.path.abspath(args.dist_dir)
+repoDir = (os.path.abspath(args.repo_dir) if args.repo_dir != '' else '')
 
 # Use a temporary directory within the build directory to assemble the components
 # for the installable package.
 tmpDir = os.path.join(buildDir, 'chip-wheel-components')
 
 manifest = json.load(open(manifestFile, 'r'))
 
+commitId = '0000000000000000000000000000000000000000'
+branch = ''
```

Comment:
Not that it's a big deal (and I already approved), but I find branch to be not so relevant for this sort of thing.  git-describe gives you your tag+delta, commit hash gives something exact.  But branch could be anything depending on what's around locally.  Maybe others disagree though.

Reply:
branch should be helpful when build / test locally since the user may identify the features it has without searching for the detailed commit id, I wonder if there are other reasons not to include branch name in version info?

Reply:
It's not a big deal.  I guess my critique is mostly relevant in product-phase version captures.  For developers, you make a good and fair point that branch names _are_ relevant.